coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -0,0 +1,36 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
+    def __init__(self, client):
+        """Inits FirebaseClientWrapper
+
+        Args:
+            Firestore client (google.cloud.firestore.Firestore): Client for interacting with Google Cloud Firestore.
+        """
+        self._client = client
+
+    @classmethod
+    def init_client(cls, crypto_token_path, app_name):
+        """Inits Firestore client
```

Comment:
This is "Inits Firestore client" but __init__ says "Inits FirebaseClientWrapper", which suggests these are returning different types when they don't. Please use the same description in both places